content/updates/2019-11-01-cdn-domain-service-replacement.md

```diff
@@ -0,0 +1,394 @@
++++
+date = "2019-11-01"
+title = "Replacement of CDN and Custom Domain Service"
++++
+
+We're announcing today the new **name** service broker, providing the new **name** and **name** services, replacing the existing CDN Service and Custom Domain Service, respectively. 
+Due to upstream deprecations, we're forced to deprecate the existing services outside of our normal deprecation policy. Beginning today, the existing CDN Service and Custom Domain service
```

Comment:
I think a bit more of detail on the upstream deprecation might help customers, with a link referencing it. Mostly for transparency and to help alleviate any questions along that vein.